app/assets/javascripts/components/sidebar_item.es6.jsx

```diff
@@ -1,28 +1,38 @@
 class SidebarItem extends React.Component {
 
-	get styles() {
+  static get propTypes() {
+    return {
+      label: React.PropTypes.string,
+      icon: React.PropTypes.string,
+    };
+  }
+
+  get styles() {
     return {
       container: {
         position: 'relative',
         width: '100%',
         padding: 30,
-        borderStyle: 'solid',
-        borderColor: 'gray',
-        borderWidth: 1,
       },
       label: {
         position: 'relative',
-      	textAlign: 'right',
-      	fontSize: 25,
+        fontSize: 15,
```

Comment:
`fontSize` should always be specified with `"px"` and should be 12, 14, 18, 24, 30, 36, or 48. Try `"14px"` here.